### PR TITLE
Search info popup hidden on click away

### PR DESCRIPTION
**Pull request by astrovsky01**
Fixing #10383

### CODE REVIEW

**Comment by astrovsky01** [comment]
I am going to also fix the need to click twice to make the help appear here

**Comment by astrovsky01** [comment]
Fixed!

**Comment by OlegZharkov** [comment]
@galaxybot test this

File: client/src/ui/search-input.js [hunk 1]
```diff
@@ -137,13 +139,18 @@ function searchInput(parentNode, options) {
...
-            .tooltip({ placement: "bottom" })
+            .tooltip({ placement: "top", trigger: "hover" })
             .click(function () {
-                $('[data-toggle="advSearchPopover"]').popover({ html: true, container: ".history-right-panel" });
+                $('[data-toggle="advSearchPopover"]').popover({
+                    trigger: "focus",
+                    html: true,
+                    container: ".history-right-panel",
+                });
+                $('[data-toggle="advSearchPopover"]').popover("show");
```
Comment (dannon) [inline client/src/ui/search-input.js]: Can this be chained so we don't need to select twice?

File: client/src/ui/search-input.js [hunk 2]
```diff
@@ -137,13 +139,18 @@ function searchInput(parentNode, options) {
...
-                '"></span>',
+                _l("Search tips"),
+                '"></a>',
             ].join("")
         )
-            .tooltip({ placement: "bottom" })
+            .tooltip({ placement: "top", trigger: "hover" })
             .click(function () {
-                $('[data-toggle="advSearchPopover"]').popover({ html: true, container: ".history-right-panel" });
+                $('[data-toggle="advSearchPopover"]').popover({
```
Comment (OlegZharkov) [inline client/src/ui/search-input.js:148]:
```suggestion
                const advSearchPopover = $('[data-toggle="advSearchPopover"]')
                advSearchPopover.popover({
```

File: client/src/ui/search-input.js (hunk 1 above)
Comment (OlegZharkov) [inline client/src/ui/search-input.js:153]:
```suggestion
                advSearchPopover.popover("show");
```

**Review by OlegZharkov — COMMENTED** [review]
that's a great contribution. You still can improve it a bit, take a look at https://api.jquery.com/click/, there should be a way to remove ```$('[data-toggle="advSearchPopover"]')``` from click()

File: client/src/ui/search-input.js (hunk 1 above)
Comment (astrovsky01) [reply client/src/ui/search-input.js]: Not that I was finding in any of the documentation, and I tried for a while to combine the commands. Might have just missed something, though...

File: client/src/ui/search-input.js (hunk 2 above)
Comment (OlegZharkov) [inline client/src/ui/search-input.js]:
```suggestion
                $(this).popover({
```

File: client/src/ui/search-input.js [hunk 3]
```diff
@@ -137,13 +139,18 @@ function searchInput(parentNode, options) {
...
+                    trigger: "focus",
+                    html: true,
+                    container: ".history-right-panel",
+                });
```
Comment (OlegZharkov) [inline client/src/ui/search-input.js]:
```suggestion
               })
```

File: client/src/ui/search-input.js (hunk 1 above)
Comment (OlegZharkov) [inline client/src/ui/search-input.js]:
```suggestion
                .popover("show");
```

**Review by OlegZharkov — CHANGES_REQUESTED** [review]
So I meant something like that

**Comment by astrovsky01** [comment]
Thanks Oleg, got the comments before the suggested changes...

**Pull request merged**